### DISCUSSION TITLE
UTXO Lease

### DISCUSSION THREAD

**Discussion by merclyn** [Ride-The-Lightning/RTL#977] [Q&A] [answered]
Under on-chain, when viewing UTXOs there is an action called "Lease"
When selecting, it says "The UTXO will be leased for 10 minutes."

What is this?  Im on LND.

**Comment by saubyk** [comment, marked as answer]
Hi @merclyn this is a UTXO level control, which LND provides for functionality like PSBT, where UTXOs need to be encumbered for a time period, while signatures are negotiated with other key holders. Since this signature negotiation can happen out of band manually, the leasing function ensures that the UTXO is not accidentally spent in other transactions, while signatures are being sought from other parties.

PSBT has not been enabled in RTL yet, but this leasing feature is nonetheless useful, if you want to control on-chain spends/channel opens from specific UTXOs. You can lease the UTXO, which you **don't** want to spend and then attempt the on-chain spend or channel opening operation, which will only include the UTXOs available i.e. not locked down with lease.

Also keep in mind that leased UTXOs are unlocked automatically after a 10 minute period, unless spent further.

I would also suggest, you can safely ignore this feature, if you don't desire to execute fine grained control over your coins in LND.

Hope that helps. Thanks.

**Reply by merclyn** [reply]
I had a feeling that the lease feature was used for this. I had found it while looking for ways to control which UTXO's were used for both channel opening, and other onchain transactions. 
Coin control features like this are crucial for maintaining privacy on the bitcoin blockchain. 

Your reply helps, and is much appreciated.
merclyn
